docs/api/web-contents.md

```diff
@@ -1345,7 +1347,9 @@ hidden and the capturer count is non-zero. If you would like the page to stay hi
 
 This also affects the Page Visibility API.
 
-#### `contents.decrementCapturerCount([stayHidden, stayAwake])`
+**Deprecated:** This API's functionality is now handled automatically within `contents.capturePage()`.
```

Comment:
Add a link to the `breaking-changes` section that describes how to migrate, please!